composer.json

```diff
@@ -70,7 +70,7 @@
     },
     "config": {
         "platform": {
-            "php": "7.1"
+            "php": "7.1.18"
```

Comment:
Why?

Reply:
PHP 7.1.0 was not enough to install Symfony 4.1 IIRC

Reply:
If it's not listed in Symfony's `composer.json`, I'd think it's not important enough. And we shouldn't set the platform this way anyway. It means we risk generating a `composer.lock` that only works with the very latest patch release of PHP 4.1, which I'm sure is not your intention.

Reply:
Good catch, I'll make an issue for that.